File: psgandalf-python/Day1/2.py
sum = 0
results = []
found = False
file = open('input.txt', 'r')
input = file.readlines()
while not found:
    for lines in input:
        results.append(sum)
        row = lines.strip(' ')
        sum += int(row)
        if sum in results:
            found = True
            break
print(sum)

File: psgandalf-javascript/Day1/2.js
const fs = require('fs')
const data = fs.readFileSync('input.txt', 'utf8')
let array = data.split('\n');
let sum = 0
let sums = [0]
let found = false
let i

while (!found) {
    for (i = 0; i < array.length; i++) {
        sum += parseInt(array[i])
        if (sums.indexOf(sum) != -1) {
            found = true
            break;
        }
        sums.push(sum)
    }
}

console.log(sum)

